patches/chrome-android-java-res-layout-new_tab_page_layout.xml.patch

```diff
@@ -11,9 +11,175 @@ index 522b6b1bfb35b83e758995459e00847bfad1ada8..9ce74212d300394a85bcf5ca7d12167e
      xmlns:android="http://schemas.android.com/apk/res/android"
      xmlns:app="http://schemas.android.com/apk/res-auto"
      xmlns:tools="http://schemas.android.com/tools"
-@@ -71,4 +71,4 @@
-         android:layout_height="0dp"
-         android:layout_weight="1"
-         android:visibility="gone" />
+     android:id="@+id/ntp_content"
```

Comment:
Looks like you accidentally applied this as a patch. You can just leave it out and it will work automatically when your new_tab_page_layout.xml file is copied over